v0.9.0/debugging-deletes.md

```diff
@@ -0,0 +1,47 @@
+# Ark version 0.7.0 and later: issue with deleting namespaces and backups 
```

Comment:
@ncdc Do we want to keep this section around? afaik it's not relevant in 0.9.0 since the finalizers get stripped.

Reply:
I'm fine removing it from 0.9+.